scripts/qa_crowbarsetup.sh

```diff
@@ -92,8 +92,6 @@ export want_postgresql=${want_postgresql:-1}
 [ -z "$want_test_updates" -a -n "$TESTHEAD" ] && export want_test_updates=1
 [ "$libvirt_type" = hyperv ] && export wanthyperv=1
 [ "$libvirt_type" = xen ] && export wantxenpv=1 # xenhvm is broken anyway
-# FIXME: delete this after cloud 7 is released
-iscloudver 6plus && export CROWBAR_EXPERIMENTAL=true
```

Comment:
Well, what meaning should it have?
`iscloudver 7plus` is a normal setup supported by crowbarctl. And check for `CROWBAR_EXPERIMENTA` was removed from crowbar-client code anyway, so I do not really see a point